tests/test_fake_toas.py

```diff
@@ -12,6 +12,34 @@
 from pinttestdata import datadir
 
 
+def roundtrip(toas, model):
+    with tempfile.NamedTemporaryFile("wt") as f:
+        toas.write_TOA_file(f)
+        f.flush()
+        toas2 = get_TOAs(f.name, model=model)
+    return toas2
+
+
+def test_roundtrip():
+    # test for roundtrip accuracy at high precision
+    parfile = os.path.join(datadir, "NGC6440E.par")
```

Comment:
This roundtrip test is only done with NGC6440E.par, which has CLK UTC(NIST), which isn't even a valid CLK entry for  PINT. We should do a roundtrip test with a few other par files, with different settings for CLK including TT(TAI) and TT(BIPM2019) and maybe different PLANET_SHAPIRO as well

Reply:
That's fair.  I think it failed before precisely because of the CLK value, but demonstrating that other clocks are OK is worthwhile.  I'll add that.

Reply:
ok, I think the latest push does this

Reply:
This does it for the clock/planet_shapiro combinations:
```
 ("UTC(NIST)", "Y"),
        ("UTC(NIST)", "N"),
        ("TT(BIPM2021)", "Y"),
        ("TT(BIPM2021)", "N"),
        ("TT(TAI)", "Y"),
        ("TT(TAI)", "N"),
```
happy to add others if wanted, but this already takes ~30s

Reply:
That seems sufficient! Could even remove one of the UTC(NIST) ones.

Reply:
since the NIST one was the problematic one, I think it's best to keep it